packages/tenant-process/src/services/validators.ts

```diff
@@ -152,6 +153,12 @@ export async function assertRequesterAllowed(
   }
 }
 
+export function assertRequesterIPAOrigin(authData: AuthData): void {
+  if (authData.externalId.origin !== PUBLIC_ADMINISTRATIONS_IDENTIFIER) {
```

Comment:
We can move all constants like this to a `models` package ? I see:
- PUBLIC_ADMINISTRATIONS_IDENTIFIER
- CONTRACT_AUTHORITY_PUBLIC_SERVICES_MANAGERS
- PUBLIC_SERVICES_MANAGERS

in this way it's more simple to identify possibles values and we have a single entry point is some change are requested.
For example this [PR](https://github.com/pagopa/interop-be-monorepo/pull/1080) introduce the similar check on externalId origin for the same identifier.